### ISSUE TITLE
Customizing Image cropper

### ISSUE THREAD

**Issue by mNalon** [cstefanache/angular2-img-cropper#170]
In "Customizing Image cropper" section should we use
 
`@ViewChild(ImageCropperComponent)
cropper:ImageCropperComponent;`

Instead of:

`@ViewChild('cropper', undefined) 
cropper:ImageCropperComponent;`

?

**Comment by zbarbuto** [comment]
You're right in that it's probably better in most cases, but it assumes you only have one cropper in your template. if your template has more than one cropper you have to name them as in the demo.
